resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -187,6 +189,10 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
   private val schedulerUuid: String = UUID.randomUUID().toString
   private val nextExecutorNumber = new AtomicLong()
 
+  private val mesosReviveThread =
```

Comment:
Nit here and everywhere in this class: style-wise I don't think we should use `mesos` in variables and function names in this class at all because it assumes that we're dealing with Mesos.

Suggestion: `mesosReviveThread` -> `reviveOffersExecutorService`

Reply:
Sure, I renamed this instance and those from the comments bellow too